crates/fluvio-sc/src/services/auth/basic.rs

```diff
@@ -285,29 +285,19 @@ mod test {
 
     #[test]
     fn test_action_urn_serialization() {
-        let action_urn = ActionUrn::new(
-            Action::Read,
-            Some(vec!["user1".to_string(), "user2".to_string()]),
-        );
+        let action_urn = ActionUrn::new(Action::Read, Some("user1".to_string()));
         let serialized =
             serde_json::to_string(&action_urn).expect("failed to serialize action urn");
-        assert_eq!(serialized, r#""Read:user1:user2""#);
+        assert_eq!(serialized, r#""Read:user1""#);
     }
 
     #[test]
     fn test_action_urn_deserialization() {
-        let deserialized: ActionUrn = serde_json::from_str(r#""Read:user1:user2:user3""#)
-            .expect("failed to deserialize action urn");
+        let deserialized: ActionUrn =
+            serde_json::from_str(r#""Read:user1""#).expect("failed to deserialize action urn");
```

Comment:
is this problem with rebase? why changes

Reply:
Oh, I think that I lost the last commit fix it after rebase, weird.

A fast fix here: https://github.com/infinyon/fluvio/pull/4024

Reply:
now it's fixed.